lib/adminlib.php

```diff
@@ -6746,6 +6746,145 @@ public function search($query) {
     }
 }
 
+/**
+ * Manage question banks page.
+ *
+ * @copyright  2021 Catalyst IT Australia Pty Ltd
+ * @author     Safat Shahin <safatshahin@catalyst-au.net>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class admin_page_manageqbanks extends admin_setting {
+    /**
+     * Constructor
+     */
+    public function __construct() {
+        $this->nosave = true;
+        parent::__construct('manageqbanks',
+                new lang_string('manageqbanks', 'admin'), '', '');
+    }
+
+    /**
+     * Always returns true
```

Comment:
I think the doc checker will pick this up.
But comments need to finish with fulls tops etc.